CMakeLists.txt

```diff
@@ -101,6 +101,17 @@ if (ENABLE_GOMA)
   set_compiler_launcher(ENABLE_GOMA gomacc)
 endif ()
 
+if(ENABLE_EXTRA_SPEED)
+	if(MSVC)
+		#enable parallel builds
+		add_compile_options("$<$<CXX_COMPILER_ID:MSVC>:/Qpar>")
```

Comment:
Drop the extra generator expression. You're already within a block wrapper w/if(MSVC).

Reply:
remove this line as well, se comment above